src/crypto/SubtleCryptoProvider.ts

```diff
@@ -1,13 +1,13 @@
-'use strict';
-
-const CryptoProvider = require('./CryptoProvider');
+import CryptoProvider = require('./CryptoProvider');
 
 /**
  * `CryptoProvider which uses the SubtleCrypto interface of the Web Crypto API.
  *
  * This only supports asynchronous operations.
  */
 class SubtleCryptoProvider extends CryptoProvider {
+  subtleCrypto: any;
```

Comment:
```suggestion
  subtleCrypto: Crypto;
```

Reply:
This doesn't seem to work with how we use `subtleCrypto`, reverting the change for now.